include/irr/asset/ICPUTextureView.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __IRR_I_CPU_TEXTURE_VIEW_H_INCLUDED__
+#define __IRR_I_CPU_TEXTURE_VIEW_H_INCLUDED__
+
+#include "irr/asset/IAsset.h"
+#include "irr/asset/IDescriptor.h"
+
+namespace irr {
+namespace asset
+{
+
+class ICPUTextureView : public IAsset, public IDescriptor
```

Comment:
you need a common base class for `ICPUTextureView` and `IGPUTextureView` that will inherit from `IDescriptor`

Reply:
@Crisspl ?

Reply:
@Crisspl TODO.

Reply:
resolved